Source/Blazorise/Fluent/FluentSizing.cs

```diff
@@ -268,7 +300,7 @@ public string Class( IClassProvider classProvider )
         {
             void BuildClasses( ClassBuilder builder )
             {
-                if ( rules.Count > 0 )
+                if ( rules is not null && rules.Count > 0 )
```

Comment:
!NullOrEmpty?

Reply:
`rules` is not a string, but a dictionary.

Reply:
Do we not have a `IsNullOrEmpty` for collections?

Reply:
I forgot we have it. haha